OrbitAsm/OrbitAsm.h

```diff
@@ -1,6 +1,10 @@
-//-----------------------------------
-// Copyright Pierric Gimmig 2013-2017
-//-----------------------------------
+/*
+ * Copyright (c) 2020 The Orbit Authors. All rights reserved.
+ * Use of this source code is governed by a BSD-style license that can be
+ * found in the LICENSE file.
+ */
```

Comment:
I think we should go with // in header files too.

Reply:
I agree, but that's not what the tool does. So, we either do this in a custom way, or we can rely now and in the future on the tool. I decided that I want to go with what tool does (similar to how we treat clang-format).

Reply:
I thought using the tool is one-time thing to do bulk update. clang-format is usually done before every commit. Is it going to be the same for the tool?

Reply:
We could add the tool to make sure all new files have the header. Not sure we want to, but independent of that, here's a tool that does this automatically in a way that should be compliant with how these headers should be added, do we really want to diverge from that?

Reply:
I don't feel too strongly about this, so can also adjust to get it consistent.

Reply:
If this is one time use - I think we should prioritize our convenience over what tool tells us. Does using // in header files make us non-compliant?

Reply:
I don't think so. Ok, let's go with // in headers as well.

Reply:
Done.